Adds SUITMap subclass to the Generic Map and implements custom colorscheme for each filters. (#7971)
sunpy/sunpy: changelog/7971.feature.rst

> Adds SUITMap subclass to the GenericMap (`sunpy/map/sources/suit.py`) and implements a custom color scheme for 11 filters of the SUIT instrument.
> For more information about the payload, visit `suit.iucaa.in <https://suit.iucaa.in>`_.

Reviewer: ```suggestion
Added "SUITMap" with a custom color scheme for 11 filters of the SUIT instrument.
For more information about the payload, visit `suit.iucaa.in <https://suit.iucaa.in>`__.
```
